x/feegrant/keeper/keeper_test.go

```diff
@@ -184,13 +182,6 @@ func (suite *KeeperTestSuite) TestUseGrantedFee() {
 			allowed: true,
 			final:   nil,
 		},
-		"expired and removed": {
```

Comment:
why removing this test case? isn't possible to set up some expired allowance with time here?

Reply:
I've added a test for `expired and removed` below. I think previous tests are wrong.